inst/htmljs/animint.js

```diff
@@ -185,6 +185,24 @@ var animint = function (to_select, json_file) {
 
   var add_geom = function (g_name, g_info) {
     // Determine if data will be an object or an array.
+    // added geom properties in steps array
+    var geom = g_info.geom;
+    var id = "plot_" + geom + "Plot";
```

Comment:
I'm not sure this is correct. can you please use a selection based on g_info.classed?
see classes of <g> elements below
![image](https://github.com/user-attachments/assets/565d49df-bf99-4373-a320-20b6f2c52c10)

Reply:
if i use g_info.classed then it shows unexpected behaviors 
![image](https://github.com/user-attachments/assets/18842b71-5a70-480e-93cb-97e90e1143e1)
![image](https://github.com/user-attachments/assets/01299d79-5c10-487b-bb32-5e7780de1dfd)

Reply:
For Geom2_vline_vlinePlot please 

* please change "Geom2_vline_vlinePlot" to the value of `g_info.classed`
* remove "Data are shown for the current selection of ." no text like this should be shown if there are no showSelected variables.
* remove "Click to change selection of: ." no text like this should be shown if there is no clickSelects variable.